pages/spacedeck/[[...roomId]].js

```diff
@@ -0,0 +1,309 @@
+                <input type="text" placeholder={t('link to sketch')} value={sketchLink} onChange={handleExistingSketch} />
+                { !validLink && sketchLink !== '' && <ErrorMessage>{ t('Make sure your link includes "{{url}}"', { url: getConfig().publicRuntimeConfig.authProviders.spacedeck.baseUrl }) }</ErrorMessage> }
+
+                <button type="submit" disabled={!sketchName || !validLink || loading}>{ loading ? <LoadingSpinnerInline inverted /> : t('Add existing sketch') }</button>
+                { errorMessage && <ErrorMessage>{ errorMessage }</ErrorMessage> }
+            </Form>);
+    };
+
+    if (!serviceSpaceId) return <LoadingSpinner />;
+
+    return (
+        <>
+            <IframeLayout.Sidebar>
+                <ServiceSubmenu
+                    title={<h2>{ getConfig().publicRuntimeConfig.authProviders.spacedeck.path }</h2>}
+                    subheadline={t('What would you like to do?')}
+                    items={[
+                        { value: 'existingSketch', actionComponentToRender: ActionExistingSketch, label: t('Add existing sketch') },
+                        { value: 'newSketch', actionComponentToRender: ActionNewSketch, label: t('Create sketch') },
+                    ]}
+                />
+                { syncingServerSketches ?
+                    <LoadingSpinner /> :
+                    <>
+                        <ServiceTable>
+                            { matrix.spaces.get(serviceSpaceId).children?.map(spacedeckRoomId => {
+                                return <ServiceLink
+                                    roomId={spacedeckRoomId}
+                                    name={matrix.rooms.get(spacedeckRoomId).name}
+                                    path={path}
+                                    selected={roomId === spacedeckRoomId}
+                                    key={spacedeckRoomId}
+                                    ref={spacedeckRoomId === roomId ? selectedPadRef : null}
+                                />;
+                            }) }
+                        </ServiceTable>
+                        { isSpacedeckServerDown && <ErrorMessage>{ t('Can\'t connect with the provided /sketch server. Please try again later.') }</ErrorMessage> }
+                    </>
+
+                }
+            </IframeLayout.Sidebar>
+            { roomId && content && (
+                <IframeLayout.IframeWrapper>
+                    <IframeLayout.IframeHeader>
+                        <h2>{ matrix.rooms.get(roomId).name }</h2>
+                        <IframeLayout.IframeHeaderButtonWrapper>
+                            <CopyToClipboard title={t('Copy sketch link to clipboard')} content={content.body} />
+                            <button title={t('Delete sketch from my library')} onClick={removeLink}>
+                                { removingLink ? <LoadingSpinnerInline /> : <Bin fill="var(--color-foreground)" /> }
```

Comment:
NAB (not a blocker): I like `isDeletingPad` like in **/write** as a variable name better... `removingLink` is making me think like... what link?!? And the `is` prefix is making it more clear that it's a boolean indicating if we're currently in the process of doing so.